install/debug.conf.template

```diff
@@ -1628,6 +1629,27 @@ ModPagespeedMessagesDomains Allow localhost
      /mod_pagespeed_example/styles/disallowed.css [L,R=301]
 </VirtualHost>
 
+<VirtualHost localhost:@@APACHE_SECONDARY_PORT@@>
+  ServerName redirecting-fetch-csp.example.com
+  DocumentRoot "@@APACHE_DOC_ROOT@@"
+  ModPagespeedFileCachePath "@@MOD_PAGESPEED_CACHE@@"
+  ModPagespeedDomain redirecting-fetch-csp.example.com:@@APACHE_SECONDARY_PORT@@
+  ModPagespeedDomain redirecting-fetch.example.com:@@APACHE_SECONDARY_PORT@@
+
+  ModPagespeedMapOriginDomain 127.0.0.1:@@APACHE_SECONDARY_PORT@@ \
+    redirecting-fetch.example.com:@@APACHE_SECONDARY_PORT@@
+  ModPagespeedMapOriginDomain 127.0.0.1:@@APACHE_SECONDARY_PORT@@ \
+    redirecting-fetch-csp.example.com:@@APACHE_SECONDARY_PORT@@
+
+  Alias "/redir_to_test" "@@APACHE_DOC_ROOT@@/mod_pagespeed_example"
+  Redirect permanent /redir_to_test/styles/blue.css http://redirecting-fetch.example.com:@@APACHE_SECONDARY_PORT@@/mod_pagespeed_example/styles/yellow.css
+
+  #Header set Content-Security-Policy "style-src http://foo.com/"
+  Header set Content-Security-Policy "default-src https:"
```

Comment:
@morlovich this CSP response header does not seem to get picked up in the scan filter [1]. When I add this directive into the html as a meta tag, this works correctly. Do you have any clue where I should start with figuring out why the response header is not available in the response headers at that point in time? 

[1] https://github.com/pagespeed/mod_pagespeed/blob/master/net/instaweb/rewriter/scan_filter.cc#L60